_rules/button-accessible-name-97a4e1.md

```diff
@@ -172,26 +208,68 @@ This off screen `button` element has no [accessible name][] because it has no co
 </html>
 ```
 
+#### Failed Example 5
+
+This `input` element with 'type' attribute in the `Image button` state has an empty [accessible name][]. The `name` attribute can not be used to provide an [accessible name][].
+
+```html
+<input type="image" name="search" src="/test-assets/shared/search-icon.svg" />
+```
+
+#### Failed Example 6
+
+This image button has an empty `alt` attribute, and no other attributes that can give it an [accessible name][].
+
+```html
+<input type="image" src="/test-assets/shared/search-icon.svg" alt="" />
+```
+
+#### Failed Example 7
+
+This image button has an `aria-labelledby` attribute, but the referenced element does not exist. This gives the button an empty [accessible name][].
+
+```html
+<input type="image" src="/test-assets/shared/search-icon.svg" aria-labelledby="non-existing" />
+```
+
```

Comment:
These examples might be hitting the corner case of the HTML AAM overloading the accessible name computation:
https://www.w3.org/TR/html-aam-1.0/#input-type-image (step 4)
Can you try and run  some tests on major browsers to check what they do?

Reply:
This example in Chrome and Safari, defaults to a submit button programmatically. If the image doesn't load, the word 'Submit' will become the button text.

Reply:
So, the description of "empty accessible name" is not correct. And these examples do not actually fail this rule (they fail the other rule given that the name is not descriptive).